zerver/tests/test_openapi.py

```diff
@@ -815,6 +818,7 @@ def test_generate_and_render_curl_with_array_example(self) -> None:
             '    --data-urlencode \'narrow=[{"operand": "Denmark", "operator": "stream"}]\' \\',
             "    --data-urlencode client_gravatar=false \\",
             "    --data-urlencode apply_markdown=false \\",
+            "    --data-urlencode target_date=2022-03-17T18:37:30.806Z \\",
```

Comment:
Is it correct to add this here for the sake of test though this doesn't match with `anchor=43`? Alternatively, we can add an `exclude` in `self.curl_example()`.